mmv1/products/bigqueryanalyticshub/Listing.yaml

```diff
@@ -151,6 +164,7 @@ properties:
     name: bigqueryDataset
     description: Shared dataset i.e. BigQuery dataset source.
     required: true
+    immutable: true
```

Comment:
Do you happen to know what's current behavior if a user is trying to update this field? Will it error out?

Making the field immutable may break users depends on how the update of the field is handled currently.

Reply:
I tried updating the field before making this change and got an error:
```
Error: Error updating Listing "projects/X/locations/US/dataExchanges/Y/listings/Z": googleapi: Error 400: The field 'bigquery_dataset' cannot be updated on this listing.
```

Reply:
Thanks for confirming! Then, it should be safe to make this change in a minor release as update is not currently supported for this field and will run into API errors in all cases, according to https://googlecloudplatform.github.io/magic-modules/develop/breaking-changes/make-a-breaking-change/#in-minor-releases